src/m365/entra/commands/m365group/m365group-user-add.ts

```diff
@@ -40,28 +44,45 @@ class EntraM365GroupUserAddCommand extends GraphCommand {
     this.#initOptions();
     this.#initValidators();
     this.#initOptionSets();
+    this.#initTypes();
   }
 
   #initTelemetry(): void {
     this.telemetry.push((args: CommandArgs) => {
       Object.assign(this.telemetryProperties, {
-        role: args.options.role,
+        role: args.options.role !== 'undefined',
         teamId: typeof args.options.teamId !== 'undefined',
-        groupId: typeof args.options.groupId !== 'undefined'
+        groupId: typeof args.options.groupId !== 'undefined',
+        teamName: typeof args.options.teamName !== 'undefined',
+        groupName: typeof args.options.groupName !== 'undefined',
+        ids: typeof args.options.ids !== 'undefined',
+        userNames: typeof args.options.userNames !== 'undefined'
```

Comment:
I know that the option is going to be deprecated, but let's also add 'username' here.